src/message.py

```diff
@@ -37,7 +38,8 @@ class Request(object):
 
     def __init__(self, user_request):
         """Initialize request by parsing info out of the request."""
-        head = user_request.split(CRLF + CRLF)[0].split(CRLF)
+        head_info = user_request.split(CRLF + CRLF)[0]
+        head = head_info.split(CRLF)
```

Comment:
Given that `CRLF` is now bound to a string that contains escaped control characters, I would expect it to have trouble with a real-life request.